src/main/java/Strings/PleaseDeleteMe.java

```diff
@@ -0,0 +1,53 @@
+package Strings;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PleaseDeleteMe {
+
+    String name ="ray maldoNADo ESTrada";
+
+    @Test
+    void ShouldReturnVowelsOnly(){
+
+        List<Character> vowels = new ArrayList<>();
+        vowels.add('A');
+        vowels.add('E');
+        vowels.add('I');
+        vowels.add('O');
+        vowels.add('U');
+
+        char[] nameChar = name.toCharArray();
+
```

Comment:
If you could use some linting to make sure all the whitespacing , blanks and aligments gets takeing care it could provide more readibility. I believe we are using checkstyle for Java Code Standard and best practices. Here it is the link with more info:  https://github.com/checkstyle/checkstyle

Reply:
@rayme11 Please see comments above